src/Routes.php

```diff
@@ -256,6 +256,7 @@ private function importEndpoints(): array
     {
         return [
             ['type' => 'store', 'verb' => 'POST', 'endpoint' => "imports/$this->resource"],
+            ['type' => 'create', 'verb' => 'GET', 'endpoint' => "import-template/$this->resource"],
```

Comment:
I like this because it would prevent ever colliding with another route

But in the future I may let the developer change the first route segment and make `imports` configurable.

Can we change this to `/imports/posts/template`?